Gameboard.java

```diff
@@ -43,10 +45,32 @@ else if (row > 10 || row < 1) {
 
  	// returns the associated column number (ex. A = 0, B = 1, etc)
  	// also adjusts for array indexing being 1 less
- 	// public int adjustColumnNumber(String column)
+ 	public int adjustColumnNumber(String column)
+    {
+        int adjustedColumn = 0;
+        
+        //iterate through letters A-J, return 
+        //adjusted column number
+        for(char letter = "A"; letter <= "J"; letter++)
+        {
+            if(column.compareToIgnoreCase(letter) == 0)
+            {
+                return adjustedColumn;
+            }
+            else
+            {
+                return null;
+            }
+            adjustedColumn++;
+        }
+    }//end method adjustColumnNumber
 
  	// returns the associated row number
  	// adjusts for array indexing being 1 less
- 	//public int adjustedRowNumber(int row)
+ 	public int adjustedRowNumber(int row)
+    {
+        int adjustedRow = row - 1;
+        return adjustedRow;
  
- }
\ No newline at end of file
+    }//end method adjustedRowNumber
+ }//end class Gameboard
```

Comment:
So lets try to have our brackets/indentation all be the same for all of our files...so fix your indentations and put the first open bracket in the same line as the method name.

the adjustedRowNumber method implementation looks correct.

The adjustcolumnNumber method is a little off I think...First of all, i messed up. the parameter should be a char type, not a string (since we'll only ever have a-j)
All that method is supposed to do is take a char and return an int. I think the easier way to implement this is with switch statements. look them up and let me know if you have trouble implementing it.